### PR TITLE
[SWPWA-332] Drop-down attribute multi-line support

### PR DESCRIPTION
**Pull request by robertsbriedis**

### CODE REVIEW

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4700026&bid=15705796)

File: src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss [hunk 1]
```diff
@@ -184,7 +188,7 @@
         }
 
         @include after-mobile {
-            padding: 0;
+            padding: 0.5rem 0;
```
Comment (Danpiel) [inline src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss]: Codacy Issue found: [Unexpected leading zero (number-leading-zero)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4700026)

File: src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss [hunk 2]
```diff
@@ -161,9 +162,12 @@
                 padding-left: 1rem;
                 padding-bottom: 0;
                 order: 1;
+                text-align: left;
 
                 ~ label {
                     margin: 0;
+                    min-width: 1.5rem;
+                    top: 0.3rem;
```
Comment (Danpiel) [inline src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss]: Codacy Issue found: [Unexpected leading zero (number-leading-zero)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4700026)

**Comment by alfredsgenkins** [comment]
Can you please add screenshots of how it looks like?

File: src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss [hunk 3]
```diff
@@ -161,9 +162,12 @@
                 padding-left: 1rem;
                 padding-bottom: 0;
                 order: 1;
+                text-align: left;
 
                 ~ label {
                     margin: 0;
+                    min-width: 1.5rem;
+                    top: .3rem;
```
Comment (alfredsgenkins) [inline src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss]: This look like a random positioning.

File: src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss [hunk 4]
```diff
@@ -140,8 +140,9 @@
     &-Text {
         display: flex;
         border: 0;
-        align-items: center;
+        align-items: baseline;
```
Comment (alfredsgenkins) [inline src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss]: Is base-line behavior intended? Can you showcase why?

**Comment by robertsbriedis** [comment]
@alfredsgenkins 
Screenshot 2019-12-19 at 10 42 38

File: src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss (hunk 3 above)
Comment (robertsbriedis) [reply src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss]: @alfredsgenkins  Combined with the baseline, we achieve the result that checkbox is aligned with text first line.

**Pull request merged**